### DISCUSSION TITLE
grafana monitoring items milvus_querynode_sq_req_latency_sum and milvus_querynode_sq_req_latency_total

### DISCUSSION THREAD

**Discussion by 1292253144** [milvus-io/milvus#36483] [Q&A and General discussion]
What are the differences between grafana monitoring items milvus_querynode_sq_req_latency_sum and milvus_querynode_sq_req_latency_total？

**Comment by yhmo** [comment]
Where did you see **milvus_querynode_sq_req_latency_total**?
Only the **milvus_querynode_sq_req_latency_sum** is mentioned in the document: https://milvus.io/docs/metrics_dashboard.md
